libindy/src/services/wallet/export_import.rs

```diff
@@ -33,7 +33,8 @@ pub(super) fn export(wallet: &Wallet, writer: &mut Write, passphrase: &str, vers
             salt: salt[..].to_vec(),
             nonce: nonce[..].to_vec(),
             chunk_size,
-        }
+        },
+        KeyDerivationMethod::RAW => return Err(WalletError::CommonError(CommonError::InvalidStructure("RAW key derivation method is not acceptable".to_string())))
```

Comment:
Help me understand why raw keys for export and import are not allowed. Why would we only support key derivation?

Reply:
I believe there is no reason except that we don't have use cases for this. 

@Artemkaaas @mikelodder7 I though a bit more. I believe it is better to keep all possible options for export too to keep API consistent.